src/DoctrineMongoODMModule/Options/Configuration.php

```diff
@@ -204,7 +204,9 @@ public function getDriver()
     public function setGenerateProxies($generateProxies)
     {
         if (is_bool($generateProxies)) {
-            $generateProxies = $generateProxies ? AbstractProxyFactory::AUTOGENERATE_ALWAYS : AbstractProxyFactory::AUTOGENERATE_NEVER;
+            $generateProxies = $generateProxies ?
+            AbstractProxyFactory::AUTOGENERATE_ALWAYS :
```

Comment:
Indentation looks off here as well - should be indented more since it belongs to the assignment.